src/code/z_scene.c

```diff
@@ -503,7 +499,8 @@ void Scene_SetTransitionForNextEntrance(PlayState* play) {
     play->transitionType = ENTRANCE_INFO_START_TRANS_TYPE(gEntranceTable[entranceIndex].field);
 }
 
-void (*gSceneCmdHandlers[SCENE_CMD_ID_MAX])(PlayState*, SceneCmd*) = {
+BAD_RETURN(s32)
+(*gSceneCmdHandlers[SCENE_CMD_ID_MAX])(PlayState*, SceneCmd*) = {
     Scene_CommandPlayerEntryList,          // SCENE_CMD_ID_SPAWN_LIST
```

Comment:
yikes. the only way around this I see is a typedef to use for the array, which tbh might not be a bad idea regardless of the formatting concern

Reply:
I think this should be a typedef even without the `BAD_RETURN`, function pointers are annoying to read.
This made me realize we don't have a typedef for MM when we should

Reply:
clang-format has a setting for this unexpectedly
https://github.com/zeldaret/oot/pull/1671/commits/817eb770211deec3d3ddafb727aa64a147fc3f18

still yeah typedef +1